docs/en/get-started/single-layer-web-application.md

```diff
@@ -73,6 +73,10 @@ Here, you can select the database management systems (DBMS){{ if DB == "EF" }} a
 
 ![abp-studio-no-layers-new-solution-additional-options](images/abp-studio-no-layers-new-solution-additional-options-0.9.13.png)
 
+Configure any additional options as needed and click the *Next* button to continue. On the next screen, you can configure the modularity options for your solution:
+
+![abp-studio-no-layers-new-solution-modularity](images/abp-studio-no-layers-new-solution-modularity.png)
+
```

Comment:
If you select the _Setup as a modular solution_ option, the solution is created more ready for [modular monolith development](../tutorials/modular-crm/index.md) and allows you to add sub-modules during the solution creation phase.